src/apps/service/apps.clj

```diff
@@ -454,13 +396,21 @@
   [user job-ids params]
   (jobs/list-job-permissions (get-apps-client user) user job-ids params))
 
+(defn validate-job-sharing-request-body
+  [user sharing-requests]
+  (jobs/validate-job-sharing-request-body (get-apps-client user) user sharing-requests))
+
 (defn share-jobs
   [user sharing-requests]
-  {:sharing (jobs/share-jobs (get-apps-client user) user sharing-requests)})
+  (jobs/share-jobs (get-apps-client user) user sharing-requests))
```

Comment:
The data associated with an asynchronous task has to be a map, so the entire request body is now passed down.